### DISCUSSION TITLE
Some Kind of Joke?

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#65] [Announcements]
![JG RTX default fog example screenshot](https://user-images.githubusercontent.com/1903667/163390985-bd9971a8-95f7-4718-a06a-0cdcefcbd389.png)

### Dev Blog
This release adds some less-exciting textures, like potatoes, compost, dead coral, filler textures and a couple updates you didn't ask for. 🎉

#### Added
- Compost
- Potatoes
- Respawn anchor
- Small drip leaf
- Sculk shrieker (Temporary design)
- Sculk catalyst
- Sculk sensor (Currently not animated)
- Horn and bubble coral fans
- Dropper front vertical

#### Changed

> ![JG RTX iron bars screenshot](https://user-images.githubusercontent.com/1903667/163391285-33b355be-6ed7-41dd-b7ed-a19d48d27772.png)
> New iron bars

- Iron bars
- Acacia trapdoor
- New, original paintings
- Decent default fog 👍

**Full Changelog**: https://github.com/jasonjgardner/jg-rtx/compare/v0.24.1...v0.25.0

#### Java Version
JG RTX has been converted to work with Java shaders, thanks to Pukaia Ariki.

![Java JG RTX](https://user-images.githubusercontent.com/1903667/163392440-de487410-0dd1-45b6-b523-2cdb1dd0182e.png)

#### Block Models
I made some models with Blockbench, which I will somehow share soon.

> ![Minecraft RTX screenshot](https://user-images.githubusercontent.com/1903667/163391466-0da56209-d842-4e60-8bfc-41e1d07d354d.png)
> ![Minecraft RTX art exhibit screenshot](https://user-images.githubusercontent.com/1903667/163391674-6c4fc1b1-71d3-4c76-bf3f-1829b97157b2.png)
> ![Minecraft RTX artwork screenshot](https://user-images.githubusercontent.com/1903667/163391725-7f14967c-d4e4-4377-83f1-aac5455d618f.png)
> ![Minecraft RTX art exhibit](https://user-images.githubusercontent.com/1903667/163391773-553f5a98-1b74-484d-9c99-ce280e571ede.png)
> Exit signs and track lights at my A**RT** E**X**hibit

> ![Minecraft RTX kaleidoscope screenshot](https://user-images.githubusercontent.com/1903667/163391121-88857009-56d1-46e2-802f-1bf446112bc3.png)
> ![Minecraft RTX kaleidoscope and studio light block model screenshot](https://user-images.githubusercontent.com/1903667/163391178-85f2e61c-556b-48aa-8a3c-76ecf4006b89.png)
> Kaleidoscope block

<hr /><em>This discussion was created from the release <a href='https://github.com/jasonjgardner/jg-rtx/releases/tag/v0.25.0'>Some Kind of Joke?</a>.</em>

**Comment by NumbThw** [comment]
You should share the models through an addon

**Reply by jasonjgardner** [reply]
One problem is that the models' textures can be often 2× to 5× the size of JG RTX's 256x textures. I quickly hit the memory limit after making a handful of models. (You can see the reduced texture detail in the kaleidoscope screenshots I shared.)

![Blockbench RTX models](https://user-images.githubusercontent.com/1903667/163564771-990cceed-56aa-427b-9bc3-24f785b116d9.png)

I believe it's due to reaching the [terrain atlas texture limit](https://discord.com/channels/691547840463241267/706976296873492511/951935713509797898). I need to redo these models with a more efficient UV layout.

I may have to split these models and extra blocks into separate add-ons. What I'd _really_ like to do is make a website to let users assemble an à la carte add-on, because I love distractions.

**Reply by NumbThw** [reply]
Oh, ok.

**Reply by ghost** [reply]
When do you think the models will be available in the pack, they look really cool

**Reply by jasonjgardner** [reply]
> When do you think the models will be available in the pack, they look really cool

Thanks! Here's some progress shared in the Minecraft RTX Discord:
https://discord.com/channels/691547840463241267/830521962383802368/972520057806675968
